units/systemd-udevd.service.in

```diff
@@ -27,6 +27,8 @@ Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket systemd-udevd-v
 Restart=always
 RestartSec=0
 ExecStart={{LIBEXECDIR}}/systemd-udevd
+FileDescriptorStoreMax=512
+FileDescriptorStorePreserve=yes
```

Comment:
So I guess this is supposed to keep around the fd across switch-root? That seems great, but it'd be nice to verify in an integration test that this is actually case. I guess the easiest way to do this is to log if we reuse the fd and check for the log message in a test that boots with an initrd.